### PR TITLE
[RHACS] ROX-18793: openshift monitoring

### PR DESCRIPTION
**Pull request by janisz**
Version(s): 4.2

Issue: https://issues.redhat.com/browse/ROX-18793

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

Cherry-pick into `rhacs-docs-4.2`

### CODE REVIEW

**Comment by janisz** [comment]
@stehessel PTAL

File: configuration/monitor-acs.adoc [hunk 1]
```diff
@@ -1,14 +1,46 @@
 [id="monitor-acs"]
-= Monitoring with Prometheus
+= Monitoring ACS
```
Comment (kcarmichael08) [inline configuration/monitor-acs.adoc:2]:
```suggestion
= Monitoring {product-title-short}
```

File: configuration/monitor-acs.adoc [hunk 2]
```diff
@@ -1,14 +1,46 @@
...
 include::modules/common-attributes.adoc[]
 :context: monitor-acs
 
 toc::[]
 
+= Monitoring with OpenShift 4
```
Comment (kcarmichael08) [inline configuration/monitor-acs.adoc:8]:
```suggestion
= Monitoring with {osp} 4
```

File: configuration/monitor-acs.adoc [hunk 3]
```diff
@@ -1,14 +1,46 @@
...
+
+[role="_abstract"]
+link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[OpenShift Container Platform] includes a preconfigured,
```
Comment (kcarmichael08) [inline configuration/monitor-acs.adoc:11]:
```suggestion
link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[{ocp}] includes a preconfigured,
```

File: configuration/monitor-acs.adoc [hunk 4]
```diff
@@ -1,14 +1,46 @@
...
+preinstalled, and self-updating monitoring stack that provides monitoring for core platform components.
+{product-title-short} has support for OpenShift Monitoring via encrypted and authenticated endpoint accessible only by Prometheus Operator.
```
Comment (kcarmichael08) [inline configuration/monitor-acs.adoc:13]:
```suggestion
{product-title-short} has support for {osp} monitoring by using encrypted and authenticated endpoint accessible only by the Prometheus Operator.
```

File: configuration/monitor-acs.adoc [hunk 5]
```diff
@@ -1,14 +1,46 @@
...
+
+[IMPORTANT]
+====
+If you have previously configured monitoring with Prometheus Operator you may consider disabling it as it will be enabled by default on OpenShift 4.
```
Comment (kcarmichael08) [inline configuration/monitor-acs.adoc:17]:
```suggestion
If you have previously configured monitoring with the Prometheus Operator, consider disabling it, because it is enabled by default on {osp} 4.
```

File: configuration/monitor-acs.adoc [hunk 6]
```diff
@@ -1,14 +1,46 @@
...
+
+= Monitoring with Prometheus
+
 [role="_abstract"]
 link:https://prometheus.io/[Prometheus] is an open-source monitoring and alerting platform.
 You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number `9090` and a network policy allowing inbound connections to that port.
 
+[NOTE]
+====
+Monitoring service is unencrypted and has no authorization. It should be used only when you do not want to use OpenShift Monitoring.
```
Comment (kcarmichael08) [inline configuration/monitor-acs.adoc:41]:
```suggestion
This monitoring service is unencrypted and has no authorization. Use this only when you do not want to use {osp} monitoring.
```

File: modules/enable-openshift-monitoring-central-helm.adoc [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+[source,terminal]
+----
+$ helm upgrade -n stackrox \
+  stackrox-central-services rhacs/central-services \
+  -f <path_to_values_public.yaml>
+----
++
+[NOTE]
+====
+You can also specify configuration values using the `--set` or `--set-file` parameters. However, these options are not saved, and it requires you to manually specify all the options again whenever you make changes.
```
Comment (kcarmichael08) [inline modules/enable-openshift-monitoring-central-helm.adoc:29]:
```suggestion
You can also specify configuration values using the `--set` or `--set-file` parameters. However, these options are not saved, and you must manually specify all of the options again whenever you make changes.
```

File: _topic_map.yml [hunk 8]
```diff
@@ -165,7 +165,7 @@ Topics:
   File: generate-diagnostic-bundle
 - Name: Configuring endpoints
   File: configure-endpoints
-- Name: Monitoring with Prometheus
+- Name: Monitoring
```
Comment (kcarmichael08) [inline _topic_map.yml:168]: I am still seeing "Monitoring with Prometheus" in the left nav/TOC in the preview.

Screenshot 2023-08-08 at 1 42 21 PM

File: modules/enable-openshift-monitoring-central-operator.adoc [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+= Monitoring Central services using the {product-title-short} Operator
+
+[role="_abstract"]
+You can monitor Central services, Central and Scanner, by changing the configuration of the `Central` custom resource.
+
+.Procedure
+. On the {ocp} web console, go to the *Operators* → *Installed Operators* page.
+. Select the {product-title} Operator from the list of installed Operators.
+. Click on the *Central* tab.
+. From the list of Central instances, click on a Central instance for which you want to enable monitoring for.
```
Comment (kcarmichael08) [inline modules/enable-openshift-monitoring-central-operator.adoc:15]:
```suggestion
. From the list of Central instances, click on a Central instance for which you want to enable monitoring.
```

File: modules/enable-openshift-monitoring-central-operator.adoc [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+:_module-type: PROCEDURE
+[id="enable-monitoring-central-operator_{context}"]
+= Monitoring Central services using the {product-title-short} Operator
+
+[role="_abstract"]
+You can monitor Central services, Central and Scanner, by changing the configuration of the `Central` custom resource.
+
+.Procedure
+. On the {ocp} web console, go to the *Operators* → *Installed Operators* page.
+. Select the {product-title} Operator from the list of installed Operators.
```
Comment (kcarmichael08) [inline modules/enable-openshift-monitoring-central-operator.adoc:13]:
```suggestion
. Select the {product-title-short} Operator from the list of installed Operators.
```

File: modules/enable-openshift-monitoring-central-operator.adoc [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+. Click on the *Central* tab.
+. From the list of Central instances, click on a Central instance for which you want to enable monitoring for.
+. Click on the *YAML* tab and update the YAML configuration:
```
Comment (kcarmichael08) [inline modules/enable-openshift-monitoring-central-operator.adoc:16]:
```suggestion
. Click on the *YAML* tab and update the YAML configuration.
```

File: modules/enable-openshift-monitoring-central-operator.adoc [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+* For monitoring Central and Scanner, enable the `monitoring.openshift.enabled` configuration option for the `Central` custom resource.
```
Comment (kcarmichael08) [inline modules/enable-openshift-monitoring-central-operator.adoc:17]:
```suggestion
For monitoring Central and Scanner, enable the `monitoring.openshift.enabled` configuration option for the `Central` custom resource, as shown in the following example:
```

File: modules/enable-openshift-monitoring-central-operator.adoc [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+----
+apiVersion: platform.stackrox.io/v1alpha1
+kind: Central
+...
+spec:
+  monitoring:
+    openshift:
+      enabled: true
+...
+----
```
Comment (kcarmichael08) [inline modules/enable-openshift-monitoring-central-operator.adoc:29]: Something is wrong here - I'm not sure what - but if you look at the preview, you can see the problem. Somehow there is a bunch of other stuff pulled into the example....

Screenshot 2023-08-08 at 1 54 14 PM

File: modules/prometheus-service-monitor.adoc [hunk 14]
```diff
@@ -8,6 +8,11 @@
 [role="_abstract"]
 If you are using the Prometheus Operator, you can use a service monitor to scrape the metrics from {rh-rhacs-first}.
 
+[IMPORTANT]
+====
+Prometheus operator monitoring is enabled by default for OpenShift 4 – no additional configuration is needed.
```
Comment (kcarmichael08) [inline modules/prometheus-service-monitor.adoc:13]:
```suggestion
Prometheus Operator monitoring is enabled by default for {osp} 4. No additional configuration is needed.
```

**Review by kcarmichael08 — COMMENTED** [review]
I can't look at the HTML preview fully because of an issue with the YAML example. I think maybe the preview hasn't updated since your latest changes, so maybe try making the suggested edits and it will cause it to rebuild. I have made some suggestions to follow the Style Guides that we have to use. Also can you please use an interactive rebase to squash commits? The OpenShift doc repo requires that we have only one commit. Thank you!

File: modules/enable-openshift-monitoring-central-operator.adoc (hunk 13 above)
Comment (kcarmichael08) [reply modules/enable-openshift-monitoring-central-operator.adoc:29]: But the code looks OK, so maybe the preview didn't update?

File: _topic_map.yml [hunk 15]
```diff
@@ -165,7 +165,7 @@ Topics:
   File: generate-diagnostic-bundle
 - Name: Configuring endpoints
   File: configure-endpoints
-- Name: Monitoring with Prometheus
```
Comment (stehessel) [inline _topic_map.yml:168]: Why did you change this?

File: configuration/monitor-acs.adoc (hunk 2 above)
Comment (stehessel) [inline configuration/monitor-acs.adoc:8]: Isn't this redundant, given that we render the docs per OpenShift version?

File: configuration/monitor-acs.adoc [hunk 16]
```diff
@@ -1,14 +1,46 @@
...
+
+[role="_abstract"]
+link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[OpenShift Container Platform] includes a preconfigured,
+preinstalled, and self-updating monitoring stack that provides monitoring for core platform components.
+{product-title-short} has support for OpenShift Monitoring via encrypted and authenticated endpoint accessible only by Prometheus Operator.
+
+[IMPORTANT]
+====
+If you have previously configured monitoring with Prometheus Operator you may consider disabling it as it will be enabled by default on OpenShift 4.
+Having multiple ServiceMonitors could result in duplicating data and wrong query results.
```
Comment (stehessel) [inline configuration/monitor-acs.adoc:18]: Shouldn't we rather advice to use the OpenShfit monitoring? I'm also not sure about the warning about wrong query results, it sounds a bit too scary for what it is imo.

File: configuration/monitor-acs.adoc (hunk 6 above)
Comment (stehessel) [reply configuration/monitor-acs.adoc:41]: Maybe a bit pedantic, but what does the "service is unencrypted" mean? Isn't the service resource always a plaintext manifest? I think it's better to be more precise here and state that the service exposes an endpoint that is not encrypted by TLS.

File: modules/enable-openshift-monitoring-central-helm.adoc [hunk 17]
```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-monitoring-central-helm_{context}"]
+= Monitoring Central services using Helm charts
+
+[role="_abstract"]
+You can monitor Central services, Central and Scanner, by changing the configuration options in the `central-services` Helm chart.
```
Comment (stehessel) [inline modules/enable-openshift-monitoring-central-helm.adoc:9]: Today this option does not expose monitoring for scanner.

File: modules/enable-openshift-monitoring-central-helm.adoc (hunk 7 above)
Comment (stehessel) [reply modules/enable-openshift-monitoring-central-helm.adoc:29]: Helm does save the options, and one can use them e.g. via `--reuse-values`. In any case, I don't think we should be too specific about how Helm works.

File: modules/prometheus-service-monitor.adoc (hunk 14 above)
Comment (stehessel) [inline modules/prometheus-service-monitor.adoc:13]: What Prometheus operator? I think this is too unspecific.

File: modules/enable-openshift-monitoring-central-operator.adoc (hunk 13 above)
Comment (janisz) [reply modules/enable-openshift-monitoring-central-operator.adoc:29]: I have problem with previews are not running for me :( I run it locally with 
- https://github.com/openshift/openshift-docs/pull/63201
and added missing `---` in https://github.com/openshift/openshift-docs/pull/63050/commits/eae9d55e0f3cf007d395d1b35a51ae90977e6974

File: _topic_map.yml (hunk 8 above)
Comment (janisz) [reply _topic_map.yml:168]: Preview does not update for me :(

File: _topic_map.yml (hunk 15 above)
Comment (janisz) [reply _topic_map.yml:168]: I though the prometheus is implementation detail of OpenShift Monitoring

File: modules/prometheus-service-monitor.adoc (hunk 14 above)
Comment (janisz) [reply modules/prometheus-service-monitor.adoc]: I added this warning based on abstract. should we link to https://github.com/prometheus-operator/prometheus-operator

Comment (stehessel) [reply modules/prometheus-service-monitor.adoc]: No, I would rather link to https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html.

**Comment by stehessel** [comment]
We also have to document `monitoring.openshift.enabled` in the Helm and CRD option lists.

File: modules/prometheus-service-monitor.adoc (hunk 14 above)
Comment (janisz) [reply modules/prometheus-service-monitor.adoc]: Then, maybe we should mark this section as deprecated?

Comment (stehessel) [reply modules/prometheus-service-monitor.adoc]: It's not deprecated though, it's part of the "manual" Prometheus setup which is still supported. Just not the default or recommended use case.

Comment (stehessel) [reply modules/prometheus-service-monitor.adoc]: But I think we can remove the `IMPORTANT` message from this section, as "Prometheus operator" in this case typically does not refer to the OpenShift Prometheus operator but a custom one.

Comment (janisz) [reply modules/prometheus-service-monitor.adoc]: But if somebody have used this to monitor with OpenShift monitoring then they might look at this section as they looked before so having a note is in my opinion good hint that it's not recommended and in fact required.

**Comment by janisz** [comment]
I added but I think not in all places. For example in Operator Central installation there is no common section where i could put that? Should I create it?

**Comment by kcarmichael08** [comment]
I'm not sure but maybe the reason the preview isn't working is because the script expects one commit per PR. They will need to be squashed anyway before merging. I used a local preview and there are a few issues that I'll mark inline.

**Review by kcarmichael08 — COMMENTED** [review]
Saw a few more style and other issues noted inline.

**Comment by kcarmichael08** [comment]
Can you please try squashing commits and force pushing and seeing if that updates the preview? (We can't merge it anyway until it's squashed.) https://docs.google.com/document/u/2/d/e/2PACX-1vRLKWEMHQ3DZroxZKfTu3XcrSdREr6D3oSSayBanEprXhkA2Ciyr2SQuDTYI4aIKUiOPPIQMHgjHeh8/pub#h.rgukz7k78tb5

File: configuration/monitor-acs.adoc [hunk 18]
```diff
@@ -1,13 +1,47 @@
...
 :context: monitor-acs
 
 toc::[]
 
+== Monitoring with {osp}
+
+[role="_abstract"]
+link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[{ocp}] includes a preconfigured,
+preinstalled, and self-updating monitoring stack that provides monitoring for core platform components.
+{product-title-short} has support for {osp} monitoring by using an encrypted and authenticated endpoint accessible only by the Prometheus Operator.
```
Comment (stehessel) [inline configuration/monitor-acs.adoc:13]:
```suggestion
{product-title-short} exposes metrics to {osp} monitoring via an encrypted and authenticated endpoint.
```

While OpenShift monitoring uses Prometheus operator under the hood, it is an implementation detail. Also it's the Prometheus pod, not the Prometheus operator pod that scrapes the metrics.

File: configuration/monitor-acs.adoc [hunk 19]
```diff
@@ -1,13 +1,47 @@
...
+
+[IMPORTANT]
+====
+If you have previously configured monitoring with the Prometheus Operator,
+consider disabling it and use OpenShift Monitoring instead that is enabled by default on {osp} 4.
+Having multiple ServiceMonitors could result in duplicating data.
```
Comment (stehessel) [inline configuration/monitor-acs.adoc:19]:
```suggestion
If you have previously configured monitoring with the Prometheus Operator,
consider removing your custom ServiceMonitor. {product-title-short} ships with a pre-configured ServiceMonitor for {osp} monitoring.

Multiple ServiceMonitors may result in duplicated scraping.
```

File: configuration/monitor-acs.adoc [hunk 20]
```diff
@@ -1,13 +1,47 @@
...
+== Monitoring with custom Prometheus
+
 [role="_abstract"]
 link:https://prometheus.io/[Prometheus] is an open-source monitoring and alerting platform.
-You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number `9090` and a network policy allowing inbound connections to that port.
+You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number 9090 and a network policy allowing inbound connections to that port.
+
+[NOTE]
+====
+This monitoring service is exposes an endpoint that is not encrypted by TLS and has no authorization.
```
Comment (stehessel) [inline configuration/monitor-acs.adoc:42]:
```suggestion
This monitoring service exposes an endpoint that is not encrypted by TLS and has no authorization.
```

File: modules/central-configuration-options-operator.adoc [hunk 21]
```diff
@@ -194,4 +194,7 @@ Setting to `true` might cause issues in some environments.
 
 | Configures whether {product-title-short} should run in online or offline mode. In offline mode, automatic updates of vulnerability definitions and kernel modules are disabled.
 
+| `monitoring.openshift.enabled`
+| If you set this option to `false`, {product-title} will not set up {osp} monitoring. Defaults to `true` on {osp} 4
```
Comment (stehessel) [inline modules/central-configuration-options-operator.adoc:198]:
```suggestion
| If you set this option to `false`, {product-title} will not set up {osp} monitoring. Defaults to `true` on {osp} 4.
```

File: modules/enable-openshift-monitoring-central-helm.adoc [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+
+[role="_abstract"]
+You can monitor  Central and Central services, by changing the configuration options in the `central-services` Helm chart.
+
+.Procedure
+. Update the `values-public.yaml` configuration file with the following values:
++
+[source,yaml]
+----
+monitoring.openshift.enabled: true
```
Comment (stehessel) [inline modules/enable-openshift-monitoring-central-helm.adoc:16]: This is already the default. Do we need this section?

File: modules/enable-openshift-monitoring-central-operator.adoc [hunk 23]
```diff
@@ -0,0 +1,29 @@
...
++
+[source,yaml]
+----
+apiVersion: platform.stackrox.io/v1alpha1
+kind: Central
+...
+spec:
+  monitoring:
+    openshift:
+      enabled: true
```
Comment (stehessel) [inline modules/enable-openshift-monitoring-central-operator.adoc:27]: Same here, not sure we need this page.

File: modules/prometheus-service-monitor.adoc [hunk 24]
```diff
@@ -8,6 +8,11 @@
 [role="_abstract"]
 If you are using the Prometheus Operator, you can use a service monitor to scrape the metrics from {rh-rhacs-first}.
 
+[IMPORTANT]
+====
+Prometheus Operator monitoring is enabled by default for {osp} 4. No additional configuration is needed.
```
Comment (stehessel) [inline modules/prometheus-service-monitor.adoc:13]:
```suggestion
{osp} monitoring is enabled by default for {osp} 4. No additional configuration is needed.
```

What are you trying to say here? This section is specifically for people using a custom Prometheus operator, where they do need additional configuration. I think what you're trying to say is that they can also just use OpenShift monitoring instead, but that is not really clear from the sentence.

File: modules/secured-cluster-services-config.adoc [hunk 25]
```diff
@@ -269,6 +269,9 @@ Otherwise, you must ensure that Central can access the online probe repository h
 | `scanner.dbResources.limits.cpu`
 | The CPU limit for the Scanner DB container. Use this parameter to override the default value.
 
+| `monitoring.openshift.enabled`
+| If you set this option to `false`, {product-title} will not set up {osp} monitoring. Defaults to `true` on {osp} 4
```
Comment (stehessel) [inline modules/secured-cluster-services-config.adoc:273]:
```suggestion
| If you set this option to `false`, {product-title} will not set up {osp} monitoring. Defaults to `true` on {osp} 4.
```

File: modules/enable-openshift-monitoring-central-helm.adoc (hunk 22 above)
Comment (janisz) [reply modules/enable-openshift-monitoring-central-helm.adoc:16]: I think we should keep it. It's easier to reason how to disable it if you see how it's enabled.

File: modules/prometheus-service-monitor.adoc (hunk 24 above)
Comment (janisz) [reply modules/prometheus-service-monitor.adoc:13]: OK, how we can improve this sentence?

File: modules/enable-openshift-monitoring-central-operator.adoc [hunk 26]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-openshift-monitoring-central-operator_{context}"]
+= Monitoring Central services by using the {product-title-short} Operator
+
+[role="_abstract"]
+You can monitor Central services (Central and Scanner) by changing the configuration of the `Central` custom resource.
```
Comment (stehessel) [inline modules/enable-openshift-monitoring-central-operator.adoc:9]:
```suggestion
You can monitor Central services by changing the configuration of the `Central` custom resource.
```

File: configuration/monitor-acs.adoc [hunk 27]
```diff
@@ -1,13 +1,48 @@
...
+[role="_abstract"]
+link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[{ocp}] includes a preconfigured,
+preinstalled, and self-updating monitoring stack that provides monitoring for core platform components.
+{product-title-short} exposes metrics to {osp} monitoring via an encrypted and authenticated endpoint.
+
+[IMPORTANT]
+====
+If you have previously configured monitoring with the Prometheus Operator,
+consider removing your custom ServiceMonitor resources. {product-title-short} ships with a pre-configured ServiceMonitor for {osp} monitoring.
+
```
Comment (kcarmichael08) [inline configuration/monitor-acs.adoc:19]: remove blank line & keep next sentence with rest of admonition

**Review by kcarmichael08 — COMMENTED** [review]
LGTM, please squash commits again. I will grab the files and look at the HTML preview tomorrow.

File: configuration/monitor-acs.adoc [hunk 28]
```diff
@@ -1,13 +1,47 @@
...
-You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number `9090` and a network policy allowing inbound connections to that port.
+You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number 9090 and a network policy allowing inbound connections to that port.
+
+[NOTE]
+====
+This monitoring service exposes an endpoint that is not encrypted by TLS and has no authorization.
+Use this only when you do not want to use {osp} monitoring.
+====
 
 include::modules/enable-monitoring-central-operator.adoc[leveloffset=+1]
```
Comment (kcarmichael08) [inline configuration/monitor-acs.adoc:46]: This is actually still not resolved, sorry. This looks like an old version of the enable-openshift-monitoring-central-operator.adoc file, which is properly called on line 22. This file included here hasn't been updated and doesn't contain the latest changes. I think this include line should be deleted, and the file enable-monitoring-central-operator.adoc should be deleted.

**Review by kcarmichael08 — COMMENTED** [review]
Line 52 looks like it is also an old duplicate. See the screenshot below - should there be 2 sections that are almost exactly the same for both of these?  

Screenshot 2023-08-30 at 12 05 16 PM

**Comment by kcarmichael08** [comment]
Here is the HTML preview that shows the page with the duplicate sections:

https://file.rdu.redhat.com/kcarmich/prs-tomas/configuration/monitor-acs.html

File: configuration/monitor-acs.adoc (hunk 28 above)
Comment (janisz) [reply configuration/monitor-acs.adoc:46]: Ah, good catch. I added more context to it but I'm not sure if it's better now

image

Comment (janisz) [reply configuration/monitor-acs.adoc:46]: maybe we should change the "old" names and add "custom" there ?

Comment (stehessel) [reply configuration/monitor-acs.adoc:46]: I feel like we now have way too many sections. Seems very complicated for a feature that is essentially working out of the box without any interaction needed. Maybe make the `... by using` sub-sections?

File: modules/enable-openshift-monitoring-central-helm.adoc (hunk 22 above)
Comment (stehessel) [reply modules/enable-openshift-monitoring-central-helm.adoc:16]: Maybe we can condense it at least, as mentioned above I think we over-document now.

**Comment by janisz** [comment]
tried with new PR to have a preview but it's still not working https://github.com/openshift/openshift-docs/pull/64417

**Comment by kcarmichael08** [comment]
Replaced by #64452
